core/src/main/scala/com/salesforce/op/OpWorkflowModelReader.scala

```diff
@@ -224,3 +230,30 @@ class OpWorkflowModelReader(val workflowOpt: Option[OpWorkflow]) extends MLReade
   }
 
 }
+
+private object FileReader {
+
+  def loadFile(pathString: String)(implicit conf: Configuration): String = {
+    val path = new Path(pathString)
+    val fs = path.getFileSystem(conf)
+    val allFiles = fs.listFiles(path, false)
+    var partPath: Option[Path] = None
+    while (allFiles.hasNext) {
+      val p = allFiles.next().getPath
+      if (p.getName.startsWith("part")) {
```

Comment:
are we really trying to grab a random part file here. POSIX does not specify the order of file entries. If we just want to grab part-00000 just specify it.

Reply:
I merged everything into a single PR but I will update this in the master PR

Reply:
the reason I am looking for starts with is because sometimes they are compressed and sometimes not in the tests